Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json

```diff
@@ -1357,6 +1356,8 @@
     "system": false,
     "version": -1,
     "fromVersion": "6.1.0",
-    "description": "",
-    "marketplaces": ["xsoar"]
+    "marketplaces": [
+        "xsoar"
```

Comment:
we also have something for XSIAM right? should we add the same message there as well ?

Reply:
In XSIAM we have the message as I added it in the past :D

Reply:
Cool - just align the text between the layouts of xsoar and xsiam :)